### DISCUSSION TITLE
couldn't find module nocodb [Jun 24]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2533] [Discord Threads]
> 🙋‍♂️ **CA-Cuckoo**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### please state your problem  
#  
> 🙋‍♂️ **CA-Cuckoo**  
#### hi  
#  
> 🙋‍♂️ **CA-Cuckoo**  
#### I try to get a nocoDB running  
#  
> 🙋‍♂️ **CA-Cuckoo**  
#### I've a linux OS installed an node v16  
#  
> 🙋‍♂️ **CA-Cuckoo**  
#### also a PG DB in v14  
#  
> 🙋‍♂️ **CA-Cuckoo**  
#### I installed reccomend to your manual:  
#  
> 🙋‍♂️ **CA-Cuckoo**  
#### git clone https://github.com/nocodb/nocodb-seed   
#### cd nocodb-seed   
#### npm install   
#### npm start  
#  
> 🙋‍♂️ **CA-Cuckoo**  
#### when I am using NPM start, following appears:  
#  
> 🙋‍♂️ **CA-Cuckoo**  
####   
![image0](https://cdn.discordapp.com/attachments/989119184107601930/989123524067205130/unknown.png)  
#  
> 🙋‍♂️ **CA-Cuckoo**  
#### Linux is Rocky v8.5  
#  
> 🙋‍♂️ **wkw**  
#### can u run ls  
#  
> 🙋‍♂️ **CA-Cuckoo**  
####   
![image0](https://cdn.discordapp.com/attachments/989119184107601930/989124757851107408/unknown.png)  
#  
> 🙋‍♂️ **CA-Cuckoo**  
####   
![image0](https://cdn.discordapp.com/attachments/989119184107601930/989124886549135440/unknown.png)  
#  
> 🙋‍♂️ **CA-Cuckoo**  
#### If helpful I can share a Teamviewer / MS Teams Session  
#  
> 🙋‍♂️ **wkw**  
#### can we hv a google meet call instead?  
#  
> 🙋‍♂️ **CA-Cuckoo**  
#### uff must check, never done before  
#  
> 🙋‍♂️ **wkw**  
#### https://meet.google.com/gba-wnnf-vaa   
####    
#### lets try  
#  
> 🙋‍♂️ **CA-Cuckoo**  
#### request is running  
#  
> 🙋‍♂️ **wkw**  
#### i didnt see it  
#  
> 🙋‍♂️ **wkw**  
#### can u try again  
#  
> 🙋‍♂️ **wkw**  
#### i only heard a sound but didnt see the accept button  
#  
> 🙋‍♂️ **CA-Cuckoo**  
#### I did  
#  
> 🙋‍♂️ **wkw**  
#### lemme DM u  
#  
> 🙋‍♂️ **CA-Cuckoo**  
#### ok are you still available today?  
#  
> 🙋‍♂️ **wkw**  
#### check DM  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/989119184107601930)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
